forge-gui-desktop/src/test/java/forge/ai/AITest.java

```diff
@@ -0,0 +1,190 @@
+package forge.ai;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.google.common.collect.Lists;
+
+import forge.GuiDesktop;
+import forge.StaticData;
+import forge.deck.Deck;
+import forge.game.Game;
+import forge.game.GameRules;
+import forge.game.GameStage;
+import forge.game.GameType;
+import forge.game.Match;
+import forge.game.card.Card;
+import forge.game.card.CardCollectionView;
+import forge.game.card.CardFactory;
+import forge.game.phase.PhaseType;
+import forge.game.player.Player;
+import forge.game.player.RegisteredPlayer;
+import forge.game.spellability.SpellAbility;
+import forge.game.zone.ZoneType;
+import forge.gui.GuiBase;
+import forge.item.IPaperCard;
+import forge.item.PaperToken;
+import forge.localinstance.properties.ForgePreferences.FPref;
+import forge.model.FModel;
+
+public class AITest {
+    private static boolean initialized = false;
+
+    public Game resetGame() {
```

Comment:
can't believe it's been over 2+ years again already since your first attempt 😱

but I recall we've had this code duplication before :p
https://github.com/Card-Forge/forge/pull/3669/commits/3a88b7ef147b9b534e27e573caf229cd236dad32

Reply:
Oh yeah, I wanted to keep things simple so I didn't mess with the old tests. I'll go ahead and fix that up.

Reply:
Even gave it the same commit message :)